README.md

```diff
@@ -4,6 +4,10 @@ ilib-webos-loctool-javascript is a plugin for the loctool that
 allows it to read and localize javascript files. This plugins is optimized for webOS platform.
 
 ## Release Notes
+v1.4.0
+* Remove commented line before parsing. so Strings in the comments don't extract.
```

Comment:
"Remove commented lines before parsing so that strings in the comments will not be extracted."

Reply:
I update it. Thank you :)